init.d/bootmisc.in

```diff
@@ -113,7 +113,7 @@ migrate_to_run()
  		rm $src
  	elif [ ! -L $src -a -d $src ]; then
  		ebegin "Migrating $src to $dst"
- 		cp -a $src/* $dst/
+ 		cp -a $src/* $dst/ 2>/dev/null
```

Comment:
this migration code is not great, so i'm not sure how much we want to bang on it.  for example, it fails to migrate dotfiles.  rsync would make short work of this.  find could handle it, but a POSIX compliant one would be pretty messy.

if we ignore dotfiles, we could write:
```
if ! rmdir $src 2>/dev/null; then
  cp -a $src/* $dst/
  rm -rf $src
fi
```

Reply:
Yes, I like that better.

Reply:
@kaniini  I do not know of any dotfiles that would be in /var/run, so I am ok with this. Please go ahead and update the pr then I'll merge it.

Reply:
Done in 061245a.